### ISSUE TITLE
This is a collection repository so there is no mandate for making issues as such.

### ISSUE THREAD

**Issue by Mohamed-Ayaan358** [acmpesuecc/code_snips#1]
If you wish to make a PR you must ask the maintainer to raise the issue.

As this is a script collection repository you are allowed to submit any file that follows the guidelines in the readme.md file.

There will be a limit of 1 PR per individual, this can be altered later on if need be.

### Bounty points(BP):-
<hr>

+ AI/ML -> 60-80 BP

+ Python and others-> 30-50 BP

+ Web-dev -> 20-40 BP

+ These BP can be altered depending on the code's complexity, length and usefulness.

**Comment by RakshithRAcharya** [comment]
Hey @Mohamed-Ayaan358 Can I do an AI/ML notebook for this?? Is there anything specific we need to do??

**Comment by Mohamed-Ayaan358** [comment]
Sure you can, what specifically is its purpose? @RakshithRAcharya

**Comment by RakshithRAcharya** [comment]
So I am trying to train autoencoders to classify fashion MNIST Data.......Is that fine??

**Comment by Mohamed-Ayaan358** [comment]
I shall raise the issue and assign it to you and you make a PR .Yes, you can go ahead with that go in the AI/ML folder and make a folder then insert your files along with a screenshot of the output if possible.

@RakshithRAcharya , you can go to the issue now.

**Comment by Smuzzy-waiii** [comment]
Can i submit a example vpython script

**Comment by mukunddeepak** [comment]
Please elaborate on what visual you are going to work on so that i can assign the issue and please make sure it is original work too! Note that plagiarized work will be deemed invalid

> Can i submit a example vpython script

**Comment by Smuzzy-waiii** [comment]
> Please elaborate on what visual you are going to work on so that i can assign the issue and please make sure it is original work too! Note that plagiarized work will be deemed invalid
> 
> > Can i submit a example vpython script

A Simple Animation of a ball bouncing in a box meant to serve as self explanatory teaching code for people who have never used vpython b4

**Comment by mukunddeepak** [comment]
Noted! Will add issue.

**Comment by Mohamed-Ayaan358** [comment]
> A Simple Animation of a ball bouncing in a box meant to serve as self explanatory teaching code for people who have never used vpython b4

Does the code have comments to explain the values as well the lines of code used to produce the output?

**Comment by Smuzzy-waiii** [comment]
yes

**Comment by mukunddeepak** [comment]
please leave a comment on issue #5 

> yes

**Comment by Vishwa-Mehta** [comment]
I'd like to add python implementations for checking valid moves for chess pieces.

**Comment by Mohamed-Ayaan358** [comment]
> I'd like to add python implementations for checking valid moves for chess pieces.

What would be the use of this script and where can it be inserted?

**Comment by Vishwa-Mehta** [comment]
> What would be the use of this script and where can it be inserted?

Pretty specific to chess so, can be inserted if one is trying to code for the entire game.
Was kind of an impromptu idea to be honest :)

**Comment by Omicron02** [comment]
I wanna add program that simulates 2 springs with a mass in between them using vpython

**Comment by Mohamed-Ayaan358** [comment]
> Pretty specific to chess so, can be inserted if one is trying to code for the entire game. Was kind of an impromptu idea to be honest :)

I am sorry but that is far too specific

@Vishwa-Mehta

**Comment by Mohamed-Ayaan358** [comment]
> I wanna add program that simulates 2 springs with a mass in between them using vpython.

Is it possible if you come up with a more unique example as to how it differs from the current issue number #5

**Comment by Omicron02** [comment]
> > I wanna add program that simulates 2 springs with a mass in between them using vpython.
> 
> Is it possible if you come up with a more unique example as to how it differs from the current issue number #5

Im sorry I didn't get you. Is it that the problem needs to be more unique?

**Comment by Mohamed-Ayaan358** [comment]
> I'm sorry I didn't get you.

someone has already raised an issue for a simple example of vpython, this topic has been done.

**Comment by Omicron02** [comment]
I'd like to add a determinant calculator using python

**Comment by Mohamed-Ayaan358** [comment]
> I'd like to add a determinant calculator using python

Can you tell me where this code can be inserted to help other codes?

**Comment by anirudhlakhotia** [comment]
I'd like to  add a password strength classifier using ML

**Comment by Omicron02** [comment]
> > I'd like to add a determinant calculator using python
> 
> Can you tell me where this code can be inserted to help other codes?

Determinants can be used in animations as they're essentially a measure of the increase in spatial area. It finds applications in linear transformations. Plus it is used when writing a program to find the values of n variables when given n equations pertaining to n spatial dimensions.

**Comment by Mohamed-Ayaan358** [comment]
Allright @Omicron02 raise an issue then raise a PR we will have a final review on the product

**Comment by Vishwa-Mehta** [comment]
> > Pretty specific to chess so, can be inserted if one is trying to code for the entire game. Was kind of an impromptu idea to be honest :)
> 
> I am sorry but that is far too specific
> 
> @Vishwa-Mehta

Okay no problem

**Comment by RakshithRAcharya** [comment]
@Mohamed-Ayaan358 Can I write a notebook for Traffic Sign Recognition??

**Comment by mukunddeepak** [comment]
Please elaborate on what you would like to work for! 

> @Mohamed-Ayaan358 Can I write a notebook for Traffic Sign Recognition??

**Comment by RakshithRAcharya** [comment]
> Please elaborate on what you would like to work for!
> 
> > @Mohamed-Ayaan358 Can I write a notebook for Traffic Sign Recognition??

AI/ML.....A notebook using German Traffic Sign Dataset...

**Comment by anirudhRowjee** [comment]
@RakshithRAcharya will you be building it from scratch? If it's a pre-existing implementation, we won't be able to award you too many points for it.

**Comment by RakshithRAcharya** [comment]
> @RakshithRAcharya will you be building it from scratch? If it's a pre-existing implementation, we won't be able to award you too many points for it.

Yes.......But I won't be able to attach the data since its huge (aroung 1.5G - 2G).......Is that fine

**Comment by anirudhRowjee** [comment]
You're welcome to open an issue, and then, a PR, but the bounty will depend on the specific implementation and the testcases you provide us with.

**Comment by RakshithRAcharya** [comment]
> You're welcome to open an issue, and then, a PR, but the bounty will depend on the specific implementation and the testcases you provide us with.

I have opened an issue...can you assign it to me??
